### ISSUE TITLE
Automation: Big Neon: Order Management: Test 37: Refund just an Order Fee from the Order

### ISSUE THREAD

**Issue by mihajlokolesar** [big-neon/bn-web#1862]
**Pre-conditions:**
1.User should have admin access to Big Neon
2.User should have created an Org with Ticket fees and Order fees
3.User should have created an event with different ticket types to account for all fee types

**User should be logged into Big Neon and within the Events Dashboard**
**steps**
1.Scroll to the event you have just published
2.Select the 3 dots for the event
3.Drop down menu list is displayed
4.Select "View Event"
5.User is redirected to the Event details page
6.Select the "Purchase Tickets" button
7.User is redirected to the Purchase page
8.Select multiples tickets for all ticket types displayed
9.Select "Continue"
10.User is redirected to Payment details page
11.Complete Payment details
12.Select "Purchase Tickets"
13.Enter phone number and select "Text me the link" button to get tickets
14.SMS sent successfully should be displayed
15.SMS and tickets should be received successfully(Check app for tickets)
16.Select the Big Neon icon
17.User is redirected to the Events Dashboard
18.Select the 3 dots for the event
19.Drop down menu list is displayed
20.Select Dashboard 
21.User is redirected to Event Dashboard
22.Select Orders > Manage Orders
23.User is redirected to the Orders page for the selected event
24.Select the Order number for the Order you would like to refund
25.Order page is displayed with all activity pertaining to the order
26.Select "Show Order details" link
27.Order details is expanded and all tickets purchased are displayed
28.Select the "Per Order fee" checkbox
29.View the refund button
30.Verify the Per order fee has been added 
31.Select Refund button
32.Issue Refund confirmation pop up is displayed
33.Select Refund reason
34.Verify refund total is correct
35.Select the Confirm button
36.Verify Refund successful pop up is displayed to the user
37.Select 'Got it"
38.Navigate back to the Order details page, verify order fee is not checkable

**Comment by mihajlokolesar** [comment]
Hi @Tas4tari ,
steps for scenario  Refund just an Order Fee from the Order from https://big-neon.ontestpad.com/script/100#158//

**Comment by mihajlokolesar** [comment]
Hi @Tas4tari , 
Please provide input regarding questions in comment. Thanks

**Comment by Tas4tari** [comment]
@mihajlokolesar  Yes this is currently not working as there is a bug logged for this. We can do this test.
